app/views/course/admin/assessments/tabs/new.html.slim

```diff
@@ -0,0 +1,9 @@
+- add_breadcrumb @category.title, course_admin_assessments_path(current_course)
```

Comment:
When creating a tab for a new category, yes. The assessment settings page is used because there isn't a `show` page for it.